### PR TITLE
Implement request card component

### PR DESCRIPTION
**Pull request by jsbuddy**
#### What does this PR do?
This PR creates reusable request card components and updates the overall UI

#### Description of task completed
- add react-icons to dependencies in package.json
- add chalk sub-dependency to resolutions in package.json to fix bug preventing app from running
- replace image icons with react icons
- modify styles to be consistent with mockup
- create request card component files
- implement request card design

#### How can it be manually tested?
- Navigate to /dashboard to see request cards

### CODE REVIEW

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -2,16 +2,27 @@
...
+  * {
+    font: inherit;
+    border: none;
+  }
+
   &__primary {
+    box-shadow: 0 10px 14px rgba(0, 0, 0, 0.1);
     background: linear-gradient(91.98deg, $purple-heart 0%, $dark-orchid 100%);
-    border-radius: rem(6px);
+    border-radius: 4px;
```
Comment (tejiri4) [inline src/components/Button/Button.scss]: use rem here.

**Comment by tejiri4** [comment]
- remove background color in most destination section on the home
- check the login and sign up page the globe image is not displaying and the form input is not align

**Comment by tejiri4** [comment]
Nice work the margin on the landing page is much for large screen. feel free to reach out if you do not understand

File: src/pages/Home/Home.scss [hunk 2]
```diff
@@ -96,66 +97,39 @@
...
-      width: rem(462px);
-      height: rem(521px);
+      width: rem(600px);
     }
   }
 
-  &__destination-header {
-    padding: rem(0) rem(100px);
+  &__destination {
+    background: #f3f3f3;
```
Comment (tejiri4) [inline src/pages/Home/Home.scss]: place the color in the assest/variables.scss

File: src/components/Button/Button.scss (hunk 1 above)
Comment (jsbuddy) [reply src/components/Button/Button.scss]: Done!

File: src/components/AppModal/AppModal.scss [hunk 3]
```diff
@@ -3,24 +3,26 @@
...
     border-radius: 50%;
-
-    img {
-      width: rem(12px);
-      height: rem(12px);
+    color: #fff;
+    cursor: pointer;
+    &:hover {
+      background: linear-gradient(90.54deg, #bb1212 0%, #be0150 100%);
+      // box-shadow: 0 4px 10px rgba(0, 0, 0, 0.07);
```
Comment (tejiri4) [inline src/components/AppModal/AppModal.scss]: do remove dead code and add this variable #bb1212 to the asset/variable file

File: src/components/Button/Button.scss [hunk 4]
```diff
@@ -2,78 +2,77 @@
...
-      $dark-orchid 100%
-    );
+    background: -webkit-linear-gradient(90.58deg, $purple-heart 0%, $dark-orchid 100%);
     background-clip: text;
     -webkit-text-fill-color: transparent;
     outline: none;
     cursor: pointer;
-    border-radius: rem(6px);
-    padding: rem(6px) rem(30px);
+    border-radius: 4px;
```
Comment (tejiri4) [inline src/components/Button/Button.scss]: do convert px to rem here.

File: src/components/DestinationCard/DestinationCard.scss [hunk 5]
```diff
@@ -1,29 +1,35 @@
 @import "~sass-rem";
 
 .destination {
-  height: rem(238px);
-  padding: rem(0) rem(20px);
-  box-sizing: border-box;
-  cursor: pointer;
+  margin-bottom: 20px;
```
Comment (tejiri4) [inline src/components/DestinationCard/DestinationCard.scss]: same here convert px to rem using rem(20px)

File: src/pages/Dashboard/index.jsx [hunk 6]
```diff
@@ -29,7 +31,24 @@ const Dashboard = () => {
           tripNum={11}
           tripName={"Total Trips"}
         />
-    </div>
+      </div>
+      <h3 className="section-title">Recent Trips</h3>
```
Comment (tejiri4) [inline src/pages/Dashboard/index.jsx]: let stick to bem methodology and put dashboard__section-title instead

File: src/components/SignUpForm/index.jsx [hunk 7]
```diff
@@ -15,13 +15,13 @@ const SignUpForm = () => {
       <div className="sign-up-form__title">
         <h1>SignUp</h1>
       </div>
-      <div className="sign-up-form__social-login">
+      <div className="col-2">
```
Comment (tejiri4) [inline src/components/SignUpForm/index.jsx]: let stick to bem methodology here and scope this too

**Comment by tejiri4** [comment]
Nice work like the new changes let try and scope the CSS so it wont affect others when trying to add a new feature

File: src/pages/Home/Home.scss [hunk 8]
```diff
@@ -56,34 +56,35 @@
     display: flex;
     justify-content: space-between;
     align-items: center;
-    min-height: auto;
-    flex-wrap: wrap;
-    padding: rem(0) rem(100px);
+    min-height: calc(100vh - 100px);
```
Comment (tejiri4) [inline src/pages/Home/Home.scss:59]: this is much when viewing on small screen, can this be reduce?

File: src/components/AppModal/AppModal.scss (hunk 3 above)
Comment (jsbuddy) [reply src/components/AppModal/AppModal.scss]: Done.

File: src/components/Button/Button.scss (hunk 4 above)
Comment (jsbuddy) [reply src/components/Button/Button.scss]: Done.

File: src/components/DestinationCard/DestinationCard.scss (hunk 5 above)
Comment (jsbuddy) [reply src/components/DestinationCard/DestinationCard.scss]: Done.

File: src/pages/Dashboard/index.jsx (hunk 6 above)
Comment (jsbuddy) [reply src/pages/Dashboard/index.jsx]: Done.

File: src/components/SignUpForm/index.jsx (hunk 7 above)
Comment (jsbuddy) [reply src/components/SignUpForm/index.jsx]: Done.

File: src/pages/Home/Home.scss (hunk 8 above)
Comment (jsbuddy) [reply src/pages/Home/Home.scss:59]: Done.

**Pull request merged**